test/performance/select_perf_test.lua

```diff
@@ -0,0 +1,176 @@
+                servers = {
+                    { instance_uuid = helpers.uuid('c', 1), alias = 's2-master' },
+                    { instance_uuid = helpers.uuid('c', 2), alias = 's2-replica' },
+                },
+            },
+            {
+                uuid = helpers.uuid('d'),
+                alias = 's-2',
+                roles = { 'customers-storage', 'crud-storage' },
+                servers = {
+                    { instance_uuid = helpers.uuid('d', 1), alias = 's3-master' },
+                    { instance_uuid = helpers.uuid('d', 2), alias = 's3-replica' },
+                },
+            }
+        },
+    })
+    g.cluster:start()
+end
+
+g.after_all = function()
+    g.cluster:stop()
+    fio.rmtree(g.cluster.datadir)
+end
+
+g.before_each(function() end)
+
+local function insert_customers(conn, id, count, timeout, report)
+    local customer = {id, box.NULL, 'David', 'Smith', 33, 'Los Angeles'}
+    local start = fiber.clock()
+
+    while (fiber.clock() - start) < timeout do
+        local ok, res, err = pcall(conn.call, conn, [[package.loaded.crud.insert]], {'customers', customer})
+        if not ok then
+            log.error('Insert error: %s', res)
+            table.insert(report.errors, res)
+        elseif err ~= nil then
+            log.error('Insert error: %s', err)
+            table.insert(report.errors, err)
+        else
+            report.count = report.count + 1
+        end
+        customer[1] = customer[1] + count
+    end
+end
+
+local function select_customers(conn, id, timeout, report)
+    local start = fiber.clock()
+    local ok, err = pcall(function()
+        while (fiber.clock() - start) < timeout do
+            local _, err = conn:call([[package.loaded.crud.select]], {'customers', {{'>', 'id', id}}, {first = 10}})
+            if err ~= nil then
+                log.error(err)
+                table.insert(report.errors, err)
+            else
+                report.count = report.count + 1
+            end
+        end
+    end)
+    if not ok then
+        table.insert(report.errors, err)
+        log.error(err)
+    end
+end
+
+g.test_insert = function()
+    local timeout = 30
+    local fiber_count = 600
+    local connection_count = 10
+    local connections = {}
+
+    local server = g.cluster.main_server
+    server.net_box:eval([[require('crud')]])
+    for _ = 1, connection_count do
+        local c = net_box:connect(server.net_box_uri, server.net_box_credentials)
+        assert(c)
+        table.insert(connections, c)
+    end
+
+    local fibers = {}
+    local report = {errors = {}, count = 0}
+    for id = 1, fiber_count do
+        local conn_id = id % connection_count + 1
+        local conn = connections[conn_id]
+        local f = fiber.new(insert_customers, conn, id, fiber_count, timeout, report)
+        f:set_joinable(true)
+        table.insert(fibers, f)
+    end
+
+    for i = 1, fiber_count do
+        fibers[i]:join()
+    end
+
+    log.error('INSERT')
```

Comment:
Do we need `error` log level here? Why log.info is not enough?

Reply:
As the highest log level it will be printed in console anyway.
"Info" could be lower than current log_level in some cases.

Currently it's not about errors and something similar. It's an approach when we don't lost this messages.